src/scripts/main.js

```diff
@@ -1,11 +1,26 @@
 'use strict';
 
 function waitFor(element, eventName) {
-// write your code here
+  return new Promise((resolve, reject) => {
+    element.addEventListener(eventName, (e) => {
```

Comment:
Все ж таки, однобуквенні змінні - зло :)
```suggestion
    element.addEventListener(eventName, (domEvent) => {
```